subsys/bluetooth/controller/ll_sw/ull_adv_types.h

```diff
@@ -19,6 +19,7 @@ struct ll_adv_set {
 	uint8_t  rnd_addr[BDADDR_SIZE];
 	uint8_t  sid:4;
 	uint8_t  is_created:1;
+	uint8_t  handle;
```

Comment:
> Thinking aloud, maybe we also need a new Kconfig in combination to CONFIG_BT_HCI_RAW, a CONFIG_BT_HCI_ZEPHYR for multi-core zephyr eco system to indicate the host on the other core is Zephyr.

I'd call it HCI_TRUSTED or similar, since essentially you're saying that you trust the host to only feed you valid data.